server/app/services/ti/TrustedIntermediaryService.java

```diff
@@ -276,8 +271,12 @@ private ImmutableList<AccountModel> searchAccounts(
         .filter(
             account ->
                 ((account.newestApplicant().get().getApplicantData().getDateOfBirth().isPresent()
-                        && searchParameters.dateQuery().isPresent()
-                        && !searchParameters.dateQuery().get().isEmpty()
+                        && searchParameters.dayQuery().isPresent()
```

Comment:
Non-blocking  - Would it be better to add this check in the SearchParam class iteself? eg. if day param is empty --> make is Optional.Empty else Optional.of(day)?